src/pages/DataLibrary/components/UserLibrary/index.tsx

```diff
@@ -106,13 +116,48 @@ const DataLibrary = () => {
           />
 
           <SplitItem>
-            <Button icon={<FaUpload />} onClick={handleFileModal}>
+            <Button icon={<FaUpload />} style ={{margin:"5px"}} onClick={handleFileModal}>
               Upload a folder
             </Button>
+        <Accordion asDefinitionList>
+          <AccordionItem>
+            <AccordionToggle
+              onClick={() => {
+                onToggle('ex-toggle1');
+              }}
+              isExpanded={expanded === 'ex-toggle1'}
+              id="ex-toggle1"
+            >
+              <DiCodeBadge /> Commands
+            </AccordionToggle>
+            <AccordionContent id="ex-expand1" isHidden={expanded !== 'ex-toggle1'}>
+              <p>
+                  # upload some files
+                <h3>chrs upload one_file.txt another_file.txt</h3>
```

Comment:
My suggestion was to add a hint for `chrs download`.

A hint for `chrs upload` is less useful because we're unable to provide a hint on what arguments to pass to `chrs upload`.

Reproducing the documentation for _chrs_ in-code here is not advisable: we will probably forget to update it when _chrs_ is updated, and to the user it's an information overload.

Reply:
Okay @jennydaman thanks for the feedback.

Reply:
@jennydaman what I'll do is I will delete these commits and start fresh.